packages/now-cli/src/commands/env/pull.ts

```diff
@@ -35,10 +48,21 @@ export default async function pull(
 
   const [filename = '.env'] = args;
   const fullPath = join(process.cwd(), filename);
-  const exists = existsSync(fullPath);
   const skipConfirmation = opts['--yes'];
 
-  if (
+  let head;
+  try {
+    head = readHeadSync(fullPath, Buffer.byteLength(CONTENTS_PREFIX));
+  } catch (err) {
```

Comment:
Can this try/catch be moved into the `readHeadSync` function?

Reply:
I think it's better not to since it's not something you expect if you're used to `fs`.

Reply:
You could rename the function to something like `tryReadHead()`